rpmlb/cli.py

```diff
@@ -73,6 +73,22 @@
     '--copr-repo', '-C', metavar='COPR_REPO',
     help='Target Copr for copr builder.',
 )
+@click.option(
+    '--epel', metavar='VERSION', type=int, default=7,
```

Comment:
If `--epel` is only used for `--build koji`,  the name `--epel` confusing users?
`--koji-epel` is better?